e2e/pages/paymentHistory.page.js

```diff
@@ -0,0 +1,20 @@
+const {I} = inject();
+const config = require('../config');
+
+const paymentsLocator = '//span[text()="Processed payments"]';
+
+const buildPaymentLocator = amount => config.mockedPayment ? `${paymentsLocator}/../table//tr[.//td[text()="Success"]]` : `${paymentsLocator}/../table//tr[.//td[text()="${amount}"] and .//td[text()="Success"]]`;
```

Comment:
config.mockedPayment is true on local env and false in pipeline. On local environment we use static wiremock mappings to simulate payment sercvice, thus fee is always same regardless of what has been actually requested to be paid. For that reason, when working with mocks, fee amount is not checked